CODEOWNERS

```diff
@@ -499,7 +499,8 @@ Makefile* @cilium/build
 /pkg/container/set/ @cilium/sig-policy
 /pkg/controller @cilium/sig-agent
 /pkg/counter @cilium/sig-datapath
-/pkg/crypto/ @cilium/sig-hubble @cilium/sig-policy
+/pkg/crypto/certificatemanager @cilium/sig-policy
```

Comment:
This package seems to be mainly used around the envoy integration, so I wonder whether @cilium/envoy would be the better fitting team.

Reply:
Happy to adjust, seems @mhofstetter approves.

Reply:
Package `/pkg/crypto/certificatemanager` is only used by Envoy secret sync (SDS) - even though parts of it are used via `pkg/policy` (but this is the servicemesh aspect of policy)

Therefore i'd suggest to remove `sig-policy` and go with

```
/pkg/crypto/certificatemanager @cilium/envoy @cilium/sig-servicemesh
```

`cilium/envoy` owning SDS sync and `cilium/sig-servicemesh` the TLS aspect within the policy - with have this overlap also in other places.